doc/Customization-guide/Customizable-system-components/Owner-verifiers/index.md

```diff
@@ -17,18 +18,18 @@ To implement a custom user verifier, you need to complete the following steps.
 3. Develop the user verifier descriptor.
 4. Provision the user verifier.
 
-We recommend that you use one of the [existing user verifier ]()[implementations]() as a reference.
+We recommend that you use one of the [existing user verifier implementations](https://github.com/kaaproject/kaa/tree/master/server/verifiers) as a reference.
 
 ## Configuration schema
 
 A user verifier configuration schema is an Avro compatible schema that defines configuration parameters of the user verifier. The following parameters in the schema affect Kaa Admin UI layout.
 
 * displayName - displays the name of the field on UI
-* by\_default - displays the default value of the field on UI  
+* by_default - displays the default value of the field on UI  
   
 ```json
     {
-     "namespace": "org.kaaproject.kaa.schema.sample",
+     "namespace": "org.kaaproject.kaa.sample.verifier.config.gen",
```

Comment:
Change import statement to "org.kaaproject.kaa.sample.verifier.config.gen.CustomUserVerifierConfiguration" in 72 and  157 lines.